app/services/audit/AuditService.scala

```diff
@@ -69,33 +69,56 @@ class AuditService @Inject()(connector: AuditConnector, appConfig: AppConfig)(im
     connector.sendExtendedEvent(extendedEvent).map(handleResponse(_, auditType))
   }
 
-  private def getAuditTags(transactionName: String, path: String)(implicit hc: HeaderCarrier) =
+  def auditMessageInboxPartialRetrieved(eori: String)(implicit hc: HeaderCarrier): Future[AuditResult] = {
+    val auditType = AuditTypes.NavigateToMessages.toString
+    val extendedEvent = ExtendedDataEvent(
+      auditSource = appConfig.appName,
+      auditType = auditType,
+      detail = detailsForMessageInboxPartialRetrieved(eori),
+      tags = tagsForMessageInboxPartialRetrieved
+    )
+
+    connector.sendExtendedEvent(extendedEvent).map(handleResponse(_, auditType))
+  }
+
+  private def getAuditTags(transactionName: String, path: String)(implicit hc: HeaderCarrier): Map[String, String] =
     AuditExtensions
       .auditHeaderCarrier(hc)
       .toAuditTags(transactionName = s"export-declaration-${transactionName.toLowerCase}", path = s"customs-declare-exports/${path}")
 
-  private def handleResponse(result: AuditResult, auditType: String) = result match {
+  private def handleResponse(result: AuditResult, auditType: String): AuditResult = result match {
     case Success =>
       logger.debug(s"Exports ${auditType} audit successful")
       Success
+
     case Failure(err, _) =>
       logger.warn(s"Exports ${auditType} Audit Error, message: $err")
       Failure(err)
+
     case Disabled =>
       logger.warn(s"Auditing Disabled")
       Disabled
   }
 
-  private def getAuditDetails(userInput: JsObject)(implicit hc: HeaderCarrier) = {
+  private def getAuditDetails(userInput: JsObject)(implicit hc: HeaderCarrier): JsObject = {
     val hcAuditDetails = Json.toJson(AuditExtensions.auditHeaderCarrier(hc).toAuditDetails()).as[JsObject]
     hcAuditDetails.deepMerge(userInput)
   }
+
+  private def detailsForMessageInboxPartialRetrieved(eori: String): JsValue =
+    Json.obj("enrolment" -> "HMRC-CUS-ORG", "eoriNumber" -> eori, "tags" -> Json.obj("notificationType" -> "CDS-EXPORTS"))
+
+  private def tagsForMessageInboxPartialRetrieved(implicit hc: HeaderCarrier): Map[String, String] =
+    AuditExtensions
+      .auditHeaderCarrier(hc)
+      .toAuditTags(transactionName = "callExportPartial", path = secureMessagingConfig.fetchInbox)
```

Comment:
Interestingly, I chose to hard code this path part for SFUS as I (maybe wrongly) assumed that this was a value that CIP would be explicitly looking for in their queries (and so should not really change). Maybe we should check with Ian & CIP about this :thinking:

Reply:
I asked yesterday to Ian and he told me to use the path specific to Exports. The data in the Confluence page is just an example.